### DISCUSSION TITLE
How to support VIMEO unlisted links?

### DISCUSSION THREAD

**Discussion by hnuecke** [vidstack/player#1181] [Q&A]
Everything works nicely (once I understood vue, ts, vidstack, vscode, npm, ... better ;-) ), 
and I managed to show a public VIMEO video in setting the SRC prop; perfect!

But obviously the format of an "**unlisted link**" is not supported.

The difference is this: 
Public videos consist of **one 9 digit** hash code, while unlisted links are composed out of **two such codes** ( 9 and 10 digits long).
Example public link:        https://vimeo.com/886369655
Example unlisted link:     https://vimeo.com/886085398/5c77542a24

Is there a way to use unlisted (private) videos with Player (vimeo provider) also?

Thanks for any suggestion and the great and well architected package!

**Comment by mihar-22** [comment]
Private videos can not be embedded, see [Vimeo overview of video privacy settings](https://help.vimeo.com/hc/en-us/articles/12426199699985-Overview-of-video-privacy-settings).
